src/components/Tooltip/Tooltip.tsx

```diff
@@ -70,6 +70,10 @@ export type Props = {
    * An optional class to apply to the tooltip message element.
    */
   tooltipClassName?: string;
+  /**
+   * The z-index value of the tooltip message element.
+   */
+  zIndex?: string;
```

Comment:
Not a particularly important reason, it's just that these properties are more often strings and I thought it would be easier to pass it as `zIndex="999"` rather than `zIndex={999}`. I can convert it to a number if it can be helpful to avoid passing invalid values - as per your other comment.